File: Day-18/Count.java
public class Count{
   
    public String countAndSay(int n) {
        String s="1";
        int cnt=1;
        for(int i=1;i<n;i++)
        {
            StringBuilder sb=new StringBuilder();
            for(int j=1;j<=s.length();j++)
            {
                if(j<s.length() && s.charAt(j)==s.charAt(j-1))
                {
                    cnt++;
                }
                else{
                    sb.append(cnt).append(s.charAt(j-1));
                    cnt=1;
                }
            } 
            s=sb.toString();
        }
        return s;
    }

}

File: Day-18/Count.py
class Count:
    def countAndSay(self, n: int) -> str:
        s="1"
        for _ in range(n-1):
            res=""
            cnt=1
            for j in range(1,len(s)):
                if(s[j]==s[j-1]):
                    cnt+=1
                else:
                    res+=str(cnt)+s[j-1]
                    cnt=1
            res+=str(cnt)+s[-1]
            s=res
        return s




        